scripts/jenkins/ardana/ansible/repositories.yml

```diff
@@ -30,11 +30,6 @@
       name: SLE12SP3-SDK-Updates
 
   # Devel:Cloud:8
-  - name: Add Devel:Cloud:8:Staging
```

Comment:
Why don't we need this repo anymore?

Reply:
we maybe still need it, but we don't want to use it. if something is not on the media then it isn't going to be delivered to customers, so  we want our CI to find such issues before customers find them.

Reply:
Ok, that makes sense